templates/solutions/ai-infrastructure/index.html

```diff
@@ -0,0 +1,454 @@
+        url="https://assets.ubuntu.com/v1/5dacced3-firmus-logo.png",
+        alt="Firmus",
+        width="200",
+        height="45",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+        }}
+      </div>
+      <div class="col-6 col-medium-4">
+        <blockquote class="p-pull-quote u-no-margin u-no-padding">
+          <div class="p-section--shallow">
+            <p class="p-heading--4">
+              <i>“The level of engagement from the Canonical team was remarkable. Even before we entered into a commercial agreement, Canonical offered valuable advice around
+              OEMs and hardware choices. They were dedicated to our project from the get-go.” </i>
+            </p>
+          </div>
+          <hr class="p-rule is-muted" />
+          <cite class="p-section--shallow">
+            <span class="p-heading--5">Tim Rosenfield</span>
+            <span class="u-float-right u-text--muted">CEO and Co-Founder, Firmus</span>
+          </cite>
+          <a class="col-8 col-medium-2"
+             href="https://ubuntu.com/engage/kubernetes-openstack-case-study-firmus">Download the full case study&nbsp;&rsaquo;</a>
+        </blockquote>
+      </div>
+      <div class="p-section col-3 col-medium-4 col-start-medium-3">
+        <div class=" u-embedded-media">
+          <iframe class="u-embedded-media__element"
+                  src="https://www.youtube.com/embed/4788ZhDqLa4?si=HKuwLsSZXL5z77Sd"
+                  title="YouTube video player"
+                  frameborder="0"
+                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                  allowfullscreen></iframe>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>The #1 Linux in the cloud</h2>
+      </div>
+      <div class="col">
+        <p>
+          Thanks to its security, versatility and policy of regular updates, Ubuntu is the most popular operating system across public clouds. And the best part is: it's free. You pay only for the commercial support you need.
+        </p>
+        <p class="p-section--shallow">Get started easily on your cloud of choice with optimised and certified images.</p>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://ubuntu.com/cloud/public-cloud">Learn more about Ubuntu on public cloud&nbsp;&rsaquo;</a>
+        </p>
+        <div class="u-fixed-width">
+          <div class="p-logo-section">
+            <div class="p-logo-section__items">
+              <div class="p-logo-section__item" style="margin-right: 2rem;">
+                {{
+                image (
+                url="https://assets.ubuntu.com/v1/55c9da63-aws-logo.png",
+                alt="AWS",
+                width="50",
+                height="104",
+                hi_def=True,
+                loading="auto|lazy",
```

Comment:
```suggestion
                loading="lazy",
```
These should be either auto or lazy, usually lazy unless they're visible in the vp on load